### ISSUE TITLE
[CoE Starter Kit - BUG] SET FLOW PERMISSIONS APP GETTING Office365Users.UserProfileV2 Failed Error

### ISSUE THREAD

**Issue by mdphung** [microsoft/coe-starter-kit#7157]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

I thought this was resolved but I am now in November Release and getting this error again.

![image](https://github.com/microsoft/coe-starter-kit/assets/117682988/c16330f9-e7f2-4f3b-8de4-9931425ef737)

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.17

### What app or flow are you having the issue with?

Dont know

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by Jenefer-Monroe** [comment]
Can you please check for unmanaged layers on the app?

**Comment by mdphung** [comment]
the only one I see is Admin | Sync Template v3 (Business Process Flows)

**Comment by Jenefer-Monroe** [comment]
Can you see if you repro this bug by going to the Power Platform Admin View equivalent for this app? 


![Image](https://github.com/microsoft/coe-starter-kit/assets/89584709/aee28009-a82f-45f2-a191-05321fa1ed89)

**Comment by mdphung** [comment]
@Jenefer-Monroe There is no error on this screen.

**Comment by Jenefer-Monroe** [comment]
Agreed on that, and it is in the plans after one more product change we are waiting for. 
Thanks for letting me know you dont repro in the newer app. We are migrating to that UX so will likely not attempt further changes in that old app.

**Comment by Jenefer-Monroe** [comment]
closing out as no further action for starter kit team
